source/includes/fact-data-prep-oracle-step1.rst

```diff
@@ -0,0 +1,22 @@
+The following code creates a new Oracle service account 
+for Relational Migrator to connect to the Oracle 
+instance. Alternatively, you can use an existing Oracle 
+service account to connect to Relational Migrator with 
+the appropriate permissions.
+
+a. Create a service account:
+
+   .. code-block:: sql
+      :copyable: true
+
+      CREATE USER 'user'@'localhost' IDENTIFIED BY 'password';
+
+#. Grant select permissions to the service account:
+
+   .. code-block:: sql
+      :copyable: true
+
+      GRANT CREATE SESSION TO user;
+      GRANT SELECT_CATALOG_ROLE TO user;
+      GRANT SELECT ANY TABLE TO user;
+      GRANT SELECT ON V$DATABASE TO user;
```

Comment:
This GRANT should be changed to 
`GRANT SELECT ON V_$DATABASE TO user;`

Reply:
@ianf-mongodb, I verified we don't really need the below for snapshot jobs, if the migration user is the owner of the migrating objects (tables), but only needed when the migration user is not owner and has least permissions. 
```
GRANT SELECT_CATALOG_ROLE TO user;
GRANT SELECT ANY TABLE TO user;
```
It does not need to be SELECT ANY TABLE (which is a wider privilege) but the below for each migrating table:

`GRANT SELECT ON TABLE TO user;`

Could we add -
1)  a note similar to - "you need these if you're not the table owner" for the above 2 statements?
2) Additionally, if the migration user is not owner, then below permission is also needed for snapshotting:
`GRANT FLASHBACK ON <TABLE> TO user;`

Reply:
I have updated this step to include a query to check table owner ship and multiple code blocks for grants depending on if the service account running the sync job is owner or not.